usaspending_api/data/testing_data/endpoint_testing_data.json

```diff
@@ -525,7 +525,7 @@
             ]
```

Comment:
I'm curious about what we think the endpoint tests should be doing these days. We're definitely better about using unit tests to hit edge cases. Are these still useful as a high-level integration test? Do need to add the new endpoints here? Or is this file strictly for legacy tests, before we got more robust with application-specific unit tests?

Personally, I've found that this series of tests, while annoying to maintain, is still useful...especially when doing schema changes that have unanticipated affects.

Not something that needs addressed to merge, just curious to hear your thoughts....would be good to be explicit about what we want the endpoint tests to accomplish.